### PR TITLE
Set up for niveristand-custom-device-handbook repo

### PR DESCRIPTION
**Pull request by cchidean**
### What does this Pull Request accomplish?
Initial setup for niveristand-custom-device-handbook repo.

### Why should this Pull Request be merged?

N/A

### What testing has been done?

N/A

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
+# Contributing to niveristand-telemetry-custom-device
```
Comment (phutanu) [inline CONTRIBUTING.md:1]: Please replace here with: "niveristand-custom-device-handbook".

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+
+Contributions to niveristand-custom-device-handbook are welcome from all!
+
+niveristand-telemetry-custom-device is managed via [git](https://git-scm.com), with the canonical upstream
```
Comment (phutanu) [inline CONTRIBUTING.md:5]: Please replace here with: "niveristand-custom-device-handbook"

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+repository hosted on [GitHub](https://github.com/ni/niveristand-custom-device-handbook/).
+
+niveristand-telemetry-custom-device follows a pull-request model for development.  If you wish to
```
Comment (phutanu) [inline CONTRIBUTING.md:8]: Please replace here with: "niveristand-custom-device-handbook"

**Review by phutanu — CHANGES_REQUESTED** [review]
Please check my above comments, and after that, this one is good to go.

**Comment by cchidean** [comment]
Feedback implemented

> Please check my above comments, and after that, this one is good to go.

**Review by phutanu — APPROVED** [review]

**Pull request merged**